share/bash-completion/completions/mech-dump

```diff
@@ -0,0 +1,12 @@
+_comp_cmd_mech-dump()
+{
+    local cur
+    _init_completion
+    cur=${COMP_WORDS[COMP_CWORD]}
+
+    # shellcheck disable=SC2207
+    COMPREPLY=( $( compgen -W "$(mech-dump --completions)" -- "$cur" ) )
```

Comment:
```suggestion
    COMPREPLY=( $( compgen -W '$(mech-dump --completions)' -- "$cur" ) )
```

The argument of `-W` is subject to the shell expansion, so to avoid extra expansions, one should quote the above by `'...'`.